src/objectivec/oga_internal.mm

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "oga_internal.h"
+
+@implementation OGAInt32Span {
```

Comment:
This is to mimic `std::span` which is not available until C++ 20, also not suitable in an ObjC environment.